debug-ui/scripts/pk-solflare-to-phantom.ts

```diff
@@ -0,0 +1,64 @@
+import 'dotenv/config';
+
+import { Keypair } from '@solana/web3.js';
+import bs58 from 'bs58';
+
+const {
+  MARKET_CREATOR_PRIVATE_KEY,
+  ALICE_PRIVATE_KEY,
+  BOB_PRIVATE_KEY,
+  CHARLIE_PRIVATE_KEY,
+} = process.env;
+
+if (!MARKET_CREATOR_PRIVATE_KEY) {
+  throw new Error('MARKET_CREATOR_PRIVATE_KEY missing from env');
```

Comment:
console.warn

Reply:
rando script handles this if not set

Reply:
made them all optional. Not my style but all good.